stable_baselines3/common/logger.py

```diff
@@ -114,12 +114,21 @@ def write_sequence(self, sequence: List) -> None:
 
 
 class HumanOutputFormat(KVWriter, SeqWriter):
-    def __init__(self, filename_or_file: Union[str, TextIO]):
+    max_length: int
```

Comment:
Hmm any reasoning to add this here? For the docstring? But the __init__ also has docstring. I lean towards not using this unique syntax for one class in the whole codebase, if possible :)

Reply:
I just added this to document that `max_length` is part of the public API and it's OK for users to modify the attribute. Since it can be awkward to set the `max_length` keyword argument in `__init__` if using `build_output_format`. I guess that's implicit by it not being `_max_length`.

Good point about this type annotation style not being used elsewhere in the codebase, though. I could just make a class docstring and document it there, or add a `@property` and `@property.setter`. What do you think?

Reply:
Good to learn about new tricks like this :D.

Hmmm I would personally opt for adding a docstring/class-docstring that instructs to modify the parameter in place. `@property` stuff would be cleaner, yes, but again something that is not much used elsewhere in the code.

Reply:
OK, added a class docstring now (and updated Docker so GitLab passes again.)